front/app/components/ParticipationCTABars/NativeSurveyCTABar/index.tsx

```diff
@@ -48,8 +48,6 @@ const NativeSurveyCTABar = ({ project }: CTABarProps) => {
         hasUserParticipated ? null : (
           <Box w="100%">
             <IdeaButton
-              id="project-survey-button"
-              data-testid="e2e-project-survey-button"
               projectId={project.id}
```

Comment:
These weren't being used anywhere and there was already another CTA button with "project-survey-button", so I've just removed them :)